### ISSUE TITLE
v6.0.1 fails to run

### ISSUE THREAD

**Issue by kaelig** [Onboardbase/onboardbase-action#2]
Here's the trace:

```
Run Onboardbase/onboardbase-action@v6.0.1
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/Onboardbase/onboardbase-action/v6.0.1/dist/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:[7](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:8)7[8](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:9):27)
    at Module.require (node:internal/modules/cjs/loader:1005:1[9](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:10))
    at require (node:internal/modules/cjs/helpers:[10](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:11)2:18)
    at Object.<anonymous> (/home/runner/work/_actions/Onboardbase/onboardbase-action/v6.0.1/dist/index.js:3:14)
    at Module._compile (node:internal/modules/cjs/loader:[11](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:12)01:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:[12](https://github.com/kaelig/kaelig.fr-next/runs/6960095109?check_suite_focus=true#step:2:13))
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/Onboardbase/onboardbase-action/v6.0.1/dist/index.js'
  ]
}
```

workaround: I reverted to v0.0.8

**Comment by limistah** [comment]
This should be fixed now, and use the version: 6.0.7

Find a full usage example here: https://github.com/Onboardbase/onboardbase-action/blob/master/.github/workflows/main.yml

And the sample result here: https://github.com/Onboardbase/onboardbase-action/runs/7054678443?check_suite_focus=true


Thank you.

**Comment by kaelig** [comment]
Thank you, it worked!
